### PR TITLE
Introduce example Scalastyle and Scapegoat issues for demonstration purposes.

### PR DESCRIPTION
**Pull request by mwz**
I picked randomly a few issues and added those to one of the example projects, so we can use it for demonstration purposes (i.e. having those show up in https://sonar.sonar-scala.com). @BalmungSan, if you're happy with those I'll replicate the changes across all the other projects.

example issues

Relates to #89.

### CODE REVIEW

File: examples/sbt/single-module/src/main/scala/Example.scala [hunk 1]
```diff
@@ -1,3 +1,18 @@
 object Example {
   def sum(a: Int, b: Int) = a + b
+
+  // An example method full of horrible style issues!
+  // This is only indended for demonstration purposes.
+  // Please don't do this at home, because every time
+  // you write code like this, a cute kitten dies!
```
Comment (BalmungSan) [inline examples/sbt/single-module/src/main/scala/Example.scala]: This comment has my dying 😸

**Comment by BalmungSan** [comment]
Cool :+1:

File: examples/sbt/single-module/src/main/scala/Example.scala (hunk 1 above)
Comment (mwz) [reply examples/sbt/single-module/src/main/scala/Example.scala]: I've must have literally killed a dozen kittens by writing this code 😢

**Comment by mwz** [comment]
Also updated the sbt multi-module project - good to go if it looks good to you @BalmungSan. Mvn coming in a separate PR; not sure about gradle though :/

**Pull request merged**